blog-service/2024-04-08-platform-services.md

```diff
@@ -0,0 +1,21 @@
+---
+title: Threat Intelligence (Platform Services)
+image: https://www.sumologic.com/img/logo.svg
+keywords:
+  - platform services
+  - threat intel
+hide_table_of_contents: true
+authors:
+  - url: https://help.sumologic.com/release-notes-service/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+We’re excited to announce a new way to use threat intelligence in Sumo Logic. You can import files containing threat intelligence indicators directly into Sumo Logic to aid in security analysis. Threat intelligence indicators are individual data points about threats that are gathered from external sources about various entities such as host names, file hashes, IP addresses, and other known targets for compromise. 
```

Comment:
Modified so that the lede is in the 1st sentence, and some other suggestions. Dunno if the last sentence is too marketing-y.

```suggestion
We’re excited to introduce a new pivotal integration that enabling users to seamlessly import threat intelligence indicator files directly into Sumo Logic to aid in security analysis. Threat intelligence indicators are individual data points about threats that are gathered from external sources about various entities such as host names, file hashes, IP addresses, and other known targets for compromise. Stay ahead of emerging threats and enhance your security posture with Sumo Logic’s latest integration.
```

Reply:
@kimsauce, I changed it to this:

>We’re excited to introduce Sumo Logic Threat Intelligence, a new feature set that enables you to seamlessly import threat intelligence indicator files directly into Sumo Logic to aid in security analysis. Threat intelligence indicators are individual data points about threats that are gathered from external sources about various entities such as host names, file hashes, IP addresses, and other known targets for compromise. Sumo Logic Threat Intelligence will help you stay ahead of emerging threats and enhance your security posture.

Reply:
This sounds great!